configure.ac

```diff
@@ -0,0 +1,186 @@
+            AC_MSG_ERROR([libtoolize not installed])
+        fi
+		;;
+	*)
+		AC_MSG_ERROR([stage1 build type '${RKT_STAGE1_USR_FROM}' unsupported])
+esac
+
+AC_SUBST(RKT_STAGE1_USR_FROM)
+AC_SUBST(RKT_STAGE1_USR_LIB)
+
+## STAGE1: Systemd git path
+AC_ARG_WITH(stage1-systemd-src,
+        AS_HELP_STRING([--with-stage1-systemd-src=git-path],
+                [address to git repository of systemd, used in 'src' build mode (default: git://anongit.freedesktop.org/systemd/systemd)]),
+        [RKT_STAGE1_SYSTEMD_SRC="$withval"],
+        [RKT_STAGE1_SYSTEMD_SRC="git://anongit.freedesktop.org/systemd/systemd"])
+
+AC_SUBST(RKT_STAGE1_SYSTEMD_SRC)
+
+## STAGE1: Systemd version
+AC_ARG_WITH(stage1-systemd-version,
+        AS_HELP_STRING([--with-stage1-systemd-version=version],
+                [systemd version to build (default: v219)]),
+        [RKT_STAGE1_SYSTEMD_VER="$withval"],
+        [RKT_STAGE1_SYSTEMD_VER="v219"])
+
+AC_SUBST(RKT_STAGE1_SYSTEMD_VER)
+
+## STAGE1: linker-defined custom STAGE1 image path, default is unset
+AC_ARG_WITH(stage1-image-path,
+        AS_HELP_STRING([--with-stage1-image-path],
+                [custom stage1 image path (default: "")]),
+        [RKT_STAGE1_IMAGE="$withval"],
+        [RKT_STAGE1_IMAGE=""])
+
+RKT_STAGE1_IMAGE_FLAGS=
+# if stage1 image variable is set, add a linker flag to rkt defining the variable
+if [[ -n "$RKT_STAGE1_IMAGE" ]] ; then
+   RKT_STAGE1_IMAGE_FLAGS="-ldflags \"-X main.defaultStage1Image '${RKT_STAGE1_IMAGE}'\""
+fi
+
+AC_SUBST(RKT_STAGE1_IMAGE_FLAGS)
+
+## STAGE1 D-Bus parameters
+AC_ARG_ENABLE(stage1-dbus,
+        AS_HELP_STRING([--enable-stage1-dbus=no],
+                [enable D-Bus in stage1 (default: no)]),
+        [RKT_STAGE1_DBUS="yes"],
+        [RKT_STAGE1_DBUS="no"])
+
+AC_SUBST(RKT_STAGE1_DBUS)
+
+AC_ARG_WITH(stage1-dbus-src,
+        AS_HELP_STRING([--with-stage1-dbus-src=git-path],
+                [address to git repository of D-Bus, used in 'src' build mode (default: git://anongit.freedesktop.org/dbus/dbus)]),
+        [RKT_STAGE1_DBUS_SRC="$withval"],
+        [RKT_STAGE1_DBUS_SRC="git://anongit.freedesktop.org/dbus/dbus"])
+ 
+AC_SUBST(RKT_STAGE1_DBUS_SRC)
+
+AC_ARG_WITH(stage1-dbus-version,
+        AS_HELP_STRING([--with-stage1-dbus-version=version],
+                [D-Bus version to build (default: dbus-1.8.16)]),
+        [RKT_STAGE1_DBUS_VER="$withval"],
+        [RKT_STAGE1_DBUS_VER="dbus-1.8.16"])
+
+AC_SUBST(RKT_STAGE1_DBUS_VER)
+
+if [[ "$RKT_STAGE1_DBUS" = "yes" ]] ; then
+
+AC_MSG_RESULT([stage1 will build D-Bus])
+
+AC_SEARCH_LIBS([_nss_files_gethostbyname_r], [nss_files], [],
+               [AC_MSG_ERROR([libnss files is needed to have D-Bus working properly. Install libc package])])
+AC_SUBST(NSS_FILES_LIB, [$ac_cv_search__nss_files_gethostbyname_r])
+
+
+AC_SEARCH_LIBS([sd_listen_fds], [systemd], [],
+               [AC_MSG_ERROR([systemd dev library needed to build D-Bus from source. Install libc package])])
```

Comment:
You probably meant some different package than libc here? Like systemd development package?